app/config/api.py

```diff
@@ -15,3 +16,4 @@ def checker(request: HttpRequest) -> dict[str, bool | str]:
 
 api.add_router("/providers", provider_router)
 api.add_router("", distributions_router)
+api.add_router("", access_router)
```

Comment:
The patterns seem to differ here... maybe unifify the way we build these APIs?

Reply:
Good point, I'll set the `provider_router` also to `""` in order to align with the others.

Reply:
This will probably lead to conflicts with https://github.com/geoadmin/service-control/pull/26/files#diff-f798933a06a1a689267b37dbf7ba771d328cb5619ae1aa9517288bfdc76d3161R7